tools/dev/loki-boltdb-storage-s3/config/loki.yaml

```diff
@@ -27,10 +27,11 @@ frontend:
     log_queries_longer_than: 5s
     max_outstanding_per_tenant: 512
 frontend_worker:
-    frontend_address: query-frontend:9007
+    # frontend_address: query-frontend:9007
     grpc_client_config:
         max_send_msg_size: 1.048576e+08
     parallelism: 6
+    scheduler_address: query-scheduler:9009
```

Comment:
I wonder whether this is a duplicate of `-querier.scheduler-address` in the Docker compose file.

Reply:
-querier.scheduler-address takes precedence.